File: ProjectTwo(Python)/models/request_loan.py

class RequestLoan:

    def __init__(self, req_id=0, user_id=0, acct_id=0, loan_amount=0, loan_term="", loan_type="", loan_status=""):
        self.req_id = req_id
        self.user_id = user_id
        self.acct_id = acct_id
        self.loan_amount = loan_amount
        self.loan_term = loan_term
        self.loan_type = loan_type
        self.loan_status = loan_status

    def __repr__(self):
        return str({
            'req_id': self.req_id,
            'user_id': self.user_id,
            'acct_id': self.acct_id,
            'loan_amount': self.loan_amount,
            'loan_term': self.loan_term,
            'loan_type': self.loan_type,
            'loan_status': self.loan_status
        })

    def json(self):
        return str({
            'reqId': self.req_id,
            'userId': self.user_id,
            'acctId': self.acct_id,
            'loanAmount': self.loan_amount,
            'loanTerm': self.loan_term,
            'loanType': self.loan_type,
            'loanStatus': self.loan_status
        })

File: JavaBackend/src/main/java/models/RequestLoan.java
package models;

public class RequestLoan {

    private int reqID;
    private int userID;
    private int acctID;
    private double loanAmount;
    private String loanTerm;
    private String loanType;
    private String loanStatus;

    public RequestLoan(){}

    public RequestLoan(int reqID, int userID, int acctID, double loanAmount, String loanTerm, String loanType, String loanStatus){
        this.reqID=reqID;
        this.userID=userID;
        this.acctID=acctID;
        this.loanAmount=loanAmount;
        this.loanTerm=loanTerm;
        this.loanType=loanType;
        this.loanStatus=loanStatus;
    }

    public int getReqID() {
        return reqID;
    }

    public void setReqID(int reqID) {
        this.reqID = reqID;
    }

    public int getUserID() {
        return userID;
    }

    public void setUserID(int userID) {
        this.userID = userID;
    }

    public int getAcctID() {
        return acctID;
    }

    public void setAcctID(int acctID) {
        this.acctID = acctID;
    }

    public double getLoanAmount() {
        return loanAmount;
    }

    public void setLoanAmount(double loanAmount) {
        this.loanAmount = loanAmount;
    }

    public String getLoanTerm() {
        return loanTerm;
    }

    public void setLoanTerm(String loanTerm) {
        this.loanTerm = loanTerm;
    }

    public String getLoanType() {
        return loanType;
    }

    public void setLoanType(String loanType) {
        this.loanType = loanType;
    }

    public String getLoanStatus() {
        return loanStatus;
    }

    public void setLoanStatus(String loanStatus) {
        this.loanStatus = loanStatus;
    }
}
